util/helpers.py

```diff
@@ -7,3 +7,19 @@ def secs_to_hours(secs):
     hours, remainder = divmod(secs, 3600)
     minutes, seconds = divmod(remainder, 60)
     return '%d:%02d:%02d' % (hours, minutes, seconds)
+
+def check_ctcdecoder_version():
+    import sys
+    import os
+    import semver
+    from ds_ctcdecoder import ctc_beam_search_decoder, Scorer, __version__ as decoder_version
+
+    decoder_version_s = decoder_version.decode()
+    ds_version_s = open(os.path.join(os.path.dirname(__file__), '../VERSION')).read().strip()
+
+    rv = semver.compare(ds_version_s, decoder_version_s)
+    if rv != 0:
+        print("DeepSpeech version ({}) and CTC decoder version ({}) do not match. Please ensure matching versions are in use.".format(ds_version_s, decoder_version_s))
+        sys.exit(-1)
```

Comment:
-1? Don't exit codes need to be in `[0, 128)`?

Reply:
Let's make it 1